### PR TITLE
[ASM] - EXPANDER - 10154 Service Ownership Playbook ASM ServiceOwners  and Ranking Score Fix

### PR DESCRIPTION
**Pull request by BigEasyJ**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
[EXPANDR-10154](https://jira-dc.paloaltonetworks.com/browse/EXPANDR-10154)

## Description
Fixed an issue in the playbook where service owners where not found in the inputs for RankServiceOwners script.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by BigEasyJ** [comment]
@capanw @johnnywilkes

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
added comment in https://jira-dc.paloaltonetworks.com/browse/EXPANDR-10154

**Review by BigEasyJ — COMMENTED** [review]
> added comment in https://jira-dc.paloaltonetworks.com/browse/EXPANDR-10154 

Pushed update from comment

**Review by johnnywilkes — APPROVED** [review]
Thanks for the change

File: Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Service_Owner.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
             "fieldCalcScript": "",
             "isDefault": true,
             "isReadOnly": false,
-            "key": "rankingscore",
+            "key": "ranking_score",
```
Comment (ssokolovich) [inline Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Service_Owner.json:68]: Where this field is being used? Moreover, how can we ensure that this change won't affect current users who are mapping to this field?

**Review by ssokolovich — COMMENTED** [review]
Looks good!
Just a small comment about the col change in the grid incident field. I want to be sure that we are not breaking anything + won't impact mapping stuff for customers.

Cheers!

File: Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Service_Owner.json (hunk 1 above)
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Service_Owner.json:68]: Current users that are mapping to this field should not be getting back any data. This field is **only** set by our 
 [RankServiceOwners](https://github.com/demisto/content/tree/master/Packs/CortexAttackSurfaceManagement/Scripts/RankServiceOwners) script and more buy the backend on the tenant.

Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Service_Owner.json:68]: Also this in an ASM field and ASM PR that is currently only being populated and used by Xpanse customers in Xpanse tenants, not XSOAR at this time.

Comment (ssokolovich) [reply Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Service_Owner.json:68]: Got ya. Cool by me! 🎉

**Review by thefrieddan1 — APPROVED** [review]

**Comment by thefrieddan1** [comment]
@BigEasyJ 
There was an infrastructure change please merge from master to allow the PR build to proceed.
Thanks

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_42.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+#### Playbooks
+
+##### Cortex ASM - Service Ownership
+
+Fixed an issue in the playbook where service owners where not found in the inputs for RankServiceOwners script.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_42.md:6]:
```suggestion
Fixed an issue in the playbook where service owners where not found in the inputs for the RankServiceOwners script.
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_42.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+#### Incident Fields
+
+- **ASM - Service Owner**
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_42.md:10]:
```suggestion
**ASM - Service Owner**
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_42.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+- Fixed an issue in **ASM - Service Owner** field where the scoring key was invalid.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_42.md:12]:
```suggestion
Fixed an issue in the **ASM - Service Owner** field where the scoring key was invalid.
```

**Comment by ShirleyDenkberg** [comment]
@ssokolovich @johnnywilkes @thefrieddan1 Doc review completed.

**Pull request merged**